src/plugins/data/server/saved_objects/schemas/query.ts

```diff
@@ -30,3 +30,11 @@ export const SCHEMA_QUERY_V8_8_0 = schema.object({
     })
   ),
 });
+
+export const SCHEMA_QUERY_V8_8_0 = SCHEMA_QUERY_BASE;
+
+export const SCHEMA_QUERY_MODEL_VERSION_1 = SCHEMA_QUERY_BASE;
+
+export const SCHEMA_QUERY_MODEL_VERSION_2 = SCHEMA_QUERY_BASE.extends({
```

Comment:
We actually shouldn't need to do this because `modelVersions` is a replacement for the existing `migrations` and `schemas` props, so we'll only need to use `modelVersions` from now on. A bit more info from the `modelVersions` TSDoc comment:
```
A map of model versions associated with this type.

Model versions supersede the migrations (and schemas) APIs by exposing an unified way of describing the changes of shape or data of the type.

Model versioning is decoupled from Kibana versioning, and isolated between types.
Model versions are identified by a single numeric value, starting at `1` and without gaps.
```